openmdao/core/total_jac.py

```diff
@@ -1242,8 +1242,16 @@ def compute_totals(self):
                             varlist = '(' + ', '.join([name for name in par_deriv[key]]) + ')'
                             print('Solving color:', key, varlist)
                         else:
-                            print('Solving variable: {0} \nIn mode: {1} \n'
-                                  'Sub Indices: {2}'.format(key, mode, inds))
+                            if key == '@simul_coloring':
+                                local_inds = imeta['coloring']._local_indices(
+                                    inds=inds, mode=self.mode)
+                                print('In mode: {0}, Solving variable(s):'.format(
+                                      mode))
+                                print(*local_inds, sep='\n')
+                                pass
+                            else:
+                                print('In mode: {0}, Solving variable(s): {1}\n'
```

Comment:
the output in the coloring case is now inconsistent with the output in other cases, i.e. in the non-colored case, it has a separate line for 'Sub Indices' which we got rid of in the colored case.   I would get rid of it here as well and make the non-colored case look like the colored case.